README.md

```diff
@@ -170,4 +174,64 @@ To conduct case selection for a new site, modify the `config.yaml` file. Add in
 During curation, sites may find cases to be ineligible and may have no additional case for which to substitute the ineligible case.  To account for expected reductions in the target count, update the `adjusted` target number in the `config.yaml` file.  These numbers will then be reflected in the `Case Selection Counts` Table on Synapse (syn26228746) and used in the QA checks to verify the number of submitted cases from each site matches the adjusted target.  
 
 ### Selection of additional samples
-After initial case selection and curation, additional samples for selected cases may be submitted through the main GENIE releases.  These additional samples are collected for curators upon request.  These cohorts are defined in much the same way as other cohorts but without production targets.  See format under the `1_additional` key in the `config.yaml` file for an example.  
+After initial case selection and curation, additional samples for selected cases may be submitted through the main GENIE releases.  These additional samples are collected for curators upon request.  These cohorts are defined in much the same way as other cohorts but without production targets.  See format under the `1_additional` key in the `config.yaml` file for an example.
+
+## Nextflow development
+
+Follow instructions here for running the genie BPC pipeline nextflow workflow locally
+
+An EC2 instance is **required** to run processing and develop locally. Follow instructions using [Service-Catalog-Provisioning](https://help.sc.sageit.org/sc/Service-Catalog-Provisioning.938836322.html) to create an ec2 on service catalog. You will also want to follow the section [SSM with SSH](https://help.sc.sageit.org/sc/Service-Catalog-Provisioning.938836322.html#ServiceCatalogProvisioning-SSMwithSSH) if you want to use VS code to run/develop.
+
+For GENIE BPC, here are the specification recommendations when launching an EC2 instance:
+
+- EC2 Product: Linux with Docker
+- EC2 Instance Type: t3.2xlarge
+- Disk size: ~100 GB
+
+### Nextflow Dependencies
+
+- [Java 8 or later](https://www.java.com/en/download/)
+- [Nextflow 22.10.x or earlier](https://www.nextflow.io/docs/latest/getstarted.html#get-started)
```

Comment:
Why earlier?  This will probably change to support later versions of nextflow when we upgrade to DSL2

Reply:
Yup, my plan is to just update this once we change to DSL2